pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -194,6 +190,31 @@ func newAlertmanager(orgID int64, cfg *setting.Cfg, store store.AlertingStore, k
 	return am, nil
 }
 
+//createSilences initializes a new instance of silence.Silences with silences taken from the file by path silencesFilePath and specific metrics registerer.
```

Comment:
```suggestion
// newSilences initializes returns *silence.Silences (from the Alertmanager) with silences taken from the file by path silencesFilePath and specific metrics registerer.
```

Reply:
Using `new` is a common idiom in go to symbolise the return an of a pointer to a struct.